tests/next_tests/integration_tests/feature_tests/iterator_tests/test_implicit_fencil.py

```diff
@@ -75,7 +75,7 @@ def fun(inp0, inp1):
     run_processor(fun[dom], program_processor, inp0, inp1, out=out, offset_provider={})
 
     if validate:
-        assert np.allclose(inp0.array() + inp1.array(), out)
+        assert np.allclose(np.asarray(inp0) + np.asarray(inp1), out)
```

Comment:
```suggestion
        assert np.allclose(inp0 + inp1, out)
```
probably this should already work